src/test/java/org/sagebionetworks/web/unitclient/widget/verification/VerificationSubmissionWidgetTest.java

```diff
@@ -0,0 +1,396 @@
+		verify(mockGinInjector).getVerificationSubmissionModalViewImpl();
+		verify(mockView).setFileHandleList(any(Widget.class));
+		verify(mockView).setWikiPage(any(Widget.class));
+		verify(mockView).setPromptModal(any(Widget.class));
+		verify(mockView).setSynAlert(any(Widget.class));
+		verify(mockView).setPresenter(widget);
+	}
+
+	
+	@Test
+	public void testConfigureAsTableRow() {
+		//TODO: for ACT place
+	}
+
+	@Test
+	public void testGetVerificationSubmissionHandleUrlOpen() {
+		configureWithMockSubmission();
+		
+		String fileHandleId = "8888";
+		widget.getVerificationSubmissionHandleUrlAndOpen(fileHandleId);
+		ArgumentCaptor<String> captor = ArgumentCaptor.forClass(String.class);
+		verify(mockSynapseJSNIUtils).getFileHandleAssociationUrl(submissionId, FileHandleAssociateType.VerificationSubmission, fileHandleId);
+		verify(mockView).openWindow(captor.capture());
+	}
+	
+	@Test
+	public void testRawFileHandleUrlAndOpen() {
+		configureWithMockSubmission();
+		String fileHandleId = "982";
+		widget.getRawFileHandleUrlAndOpen(fileHandleId);
+		verify(mockSynapseJSNIUtils).getBaseFileHandleUrl();
+		verify(mockView).openWindow(anyString());
+	}
+
+	private UserProfile getPopulatedProfile() {
+		UserProfile profile = new UserProfile();
+		profile.setFirstName("Bruce");
+		profile.setLastName("Banner");
+		profile.setCompany("U.S. Government");
+		profile.setLocation("Unknown");
+		profile.setEmails(Collections.singletonList("bruce@gov.org"));
+		return profile;
+	}
+	
+	@Test
+	public void testIsPreconditionsForNewSubmissionMet() {
+		configureWithMockSubmission();
+		String orcId = "http://orcid.org/123";
+		UserProfile profile = getPopulatedProfile();
+		
+		assertTrue(widget.isPreconditionsForNewSubmissionMet(profile, orcId));
+		assertFalse(widget.isPreconditionsForNewSubmissionMet(profile, ""));
+		profile.setFirstName("");
+		assertFalse(widget.isPreconditionsForNewSubmissionMet(profile, orcId));
+		profile = getPopulatedProfile();
+		profile.setLastName("");
+		assertFalse(widget.isPreconditionsForNewSubmissionMet(profile, orcId));
+		profile = getPopulatedProfile();
+		profile.setCompany("");
+		assertFalse(widget.isPreconditionsForNewSubmissionMet(profile, orcId));
+		profile = getPopulatedProfile();
+		profile.setLocation("");
+		assertFalse(widget.isPreconditionsForNewSubmissionMet(profile, orcId));
+	}
+	
+	@Test
+	public void testShowNewVerificationSubmission() {
+		String orcId = "http://orcid.org/123";
+		UserProfile profile = getPopulatedProfile();
+		boolean isACTMember = false;
+		boolean isModal = true;
+		widget.configure(profile, orcId, isACTMember, isModal);
+		widget.show();
+		verify(mockView).setWikiPageVisible(true);
+		verify(mockView).setCancelButtonVisible(true);
+		verify(mockView).setSubmitButtonVisible(true);
+		verify(mockFileHandleList).configure(any(CallbackP.class));
+		verify(mockFileHandleList).setUploadButtonText(anyString());
+		verify(mockFileHandleList).setCanDelete(true);
```

Comment:
should verify that other button visibility is not changed (defaults to false from clear).